index.html

```diff
@@ -7,12 +7,143 @@
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+        </div>
+        </nav>
+        
+    </header>
+
+    <section class="hero">
+        <div class="hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT <br />2015</h1>
+        <p class="description">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+        <h2 class="date">2015.10.15(THU) ~ 16(FRI)</h2>
+        <p class="location">@ National Museum of Karea, Art Center Nabi and more
+        </p>
+    </div>
+    </section>
+
+    <section class="program">
+        <div class="program-wrapper">
+
+            <h2 class="program-title">Main Program</h2>
+            <hr />
+
+            <div class="all-programs">
+
+            <div class="program-card">
+                <img src="./images/program_icon_01.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">Lecture</h3>
+                <p class="program-desc">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_02.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">CC Exhibition</h3>
+                <p class="program-desc">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_03.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">Forum</h3>
+                <p class="program-desc">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_04.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">WorkShop</h3>
+                <p class="program-desc">Try creating your own work using open source license rather than just watching at it.
+                </p>
+            </div>
+
+            <div class="program-card">
+                <img src="./images/program_icon_05.svg" alt="program One" class="program-icon">
+                <h3 class="program-name">CC Ignite</h3>
+                <p class="program-desc">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+            </div>
+
+        </div>
+            <a href="about.html" class="join-btn">Join CC Global 2015</a>
+            <a href="#" class="join-btn2">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+
+    <section class="speakers">
```

Comment:
- [ ] Good job on rendering your featured speakers dynamically 👍🏽 . I just noticed that on the page load, only two featured speakers is displayed. Kindly modify your functionality so that the six featured speakers and their descriptions appear on the desktop view when the page is visited so that it matches the design you are given.  

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235222396-9285b51f-c230-401e-ba56-561db07e3b55.png)

design guildeline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235222669-a25c8eb4-9fcb-46f4-8f11-053f1986ec2d.png)